.github/workflows/ci.yml

```diff
@@ -59,8 +73,31 @@ jobs:
         if: ${{ env.MAINNET_RPC_URL != null }}
         run: pnpm -C packages/hardhat test
 
-      - name: Test foundry
-        env:
-          RPC_URL: ${{ secrets.MAINNET_RPC_URL }}
-        if: ${{ env.MAINNET_RPC_URL != null }}
-        run: pnpm -C packages/hardhat forge test -vv
+      - name: Foundry forge build
+        run: pnpm -C packages/hardhat forge-build
+
+      - name: Foundry forge test
+        run: pnpm -C packages/hardhat forge-test
+
+      - name: Slither analyze
+        id: slither
+        continue-on-error: true
+        run: pnpm -C packages/hardhat slither:ci
+
+      - name: Check slither SARIF output existence
```

Comment:
We can speed this up in the future with something like `run: [ -f .packages/hardhat/results.sarif ] && echo "file_exists=true" >> "$GITHUB_OUTPUT"`